docs/resources/logdna_view.md

```diff
@@ -108,7 +108,7 @@ _Note:_ At least one of the following properties: `apps`, `hosts`, `levels`, `qu
 
 `webhook_channel` supports the following arguments:
 
-- `bodytemplate`: _(Optional)_ JSON Object for the body of the webhook, type Map of _strings_
+- `bodytemplate`: _(Optional)_ JSON formatted string for the body of the webhook. We recommend using [`jsonencode()`](https://www.terraform.io/docs/configuration/functions/jsonencode.html) to easily convert a Terraform map into a JSON string. Type _string_
```

Comment:
@dm36, it's `bodyTemplate` instead of `bodytemplate`

Reply:
@smusali field names can't be named with camelcase in terraform. When the bodytemplate data is forwarded to the config-api it's sent as bodyTemplate https://github.com/logdna/terraform-provider-logdna/blob/main/logdna/client.go#L44:

![Screen Shot 2020-10-28 at 10 21 32 AM](https://user-images.githubusercontent.com/10788080/97472765-6f6fde80-1907-11eb-852e-86054013e59c.png)